src/apps/MobileApp/MobileClient.js

```diff
@@ -1314,7 +1314,7 @@ class MobileClient extends Component {
                     return;
                 }
 
-                this.state.virtualStreamingJanus.streamGalaxy(data.status, 4, "");
+                this.state.shidurJanus.streamGalaxy(data.status, 4, "");
```

Comment:
Maybe leave it as it was and change state property name as it in VirutalClient?

Reply:
I have updated what you like but there is no end to this...

sourceLoading vs shidurLoading
shidurJanus in VirtualStreamingMobile.
....

There are lots of them... drop in the sea... we should refactor the code, not fix one variable name.